tests/lp_files/storage.lp

```diff
@@ -2,6 +2,10 @@
 
 min 
 objective:
++4000.0 ONE_VAR_CONSTANT
++0.1 GenericStorageBlock_storage_content(storage_no_invest_1)
++0.1 GenericStorageBlock_storage_content(storage_no_invest_2)
++0.1 GenericStorageBlock_storage_content(storage_no_invest_3)
```

Comment:
Apparently there is no `period` index for the storage block. This is not a problem of this issue, but should be fixed for the sake of consistency, shouldn't it?

Reply:
As the storage content is given in absolute values, only the limit (which might change from period to period) needs to have the index. I will leave it as it is.